jcommune-performance-tests/src/test/scala/org/jtalks/jcommune/performance/tests/Login.scala

```diff
@@ -0,0 +1,44 @@
+/**
+  * Copyright (C) 2011  JTalks.org Team
+  * This library is free software; you can redistribute it and/or
+  * modify it under the terms of the GNU Lesser General Public
+  * License as published by the Free Software Foundation; either
+  * version 2.1 of the License, or (at your option) any later version.
+  * This library is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  * Lesser General Public License for more details.
+  * You should have received a copy of the GNU Lesser General Public
+  * License along with this library; if not, write to the Free Software
+  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+  */
+package org.jtalks.jcommune.performance.tests
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class Login extends Simulation{
+  val url = "http://performance.jtalks.org/jcommune"
+  val username = "admin"
+  val password = "admin"
+
+  val httpConf = http
+    .baseURL(url)
+    .acceptHeader("text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8")
+    .doNotTrackHeader("1")
+    .acceptLanguageHeader("en-US,en;q=0.5")
+    .acceptEncodingHeader("gzip, deflate")
+    .userAgentHeader("Mozilla/5.0 (Windows NT 5.1; rv:31.0) Gecko/20100101 Firefox/31.0")
+
+  val scn = scenario("BasicSimulation")
+    .exec(http("request_1")
+      .post("/jcommune/login_ajax")
+      .formParam("userName", username)
+      .formParam("password", password)
+      .formParam("_spring_security_remember_me", "on"))
+    .pause(5)
+
+  setUp(
+    scn.inject(atOnceUsers(30))
```

Comment:
Could you please change it to scn.inject(constantUsersPerSec(3) during DurationInteger(1800).seconds randomized) because we used this values in jMeter.